### PR TITLE
docs: add 3d-view endpoint doc

### PR DESCRIPTION
**Pull request by lniehaus**

### CODE REVIEW

**Comment by jarkenau** [comment]
@Anafabula unified the api docs, in the last days, your docs don't follow the new structure. Every page now hast an overview section at the beginning, https://naturerobots.github.io/HSOS-SEP-PlantMap-2022/reference/bed-plant-endpoints/#overview. And the headers after that use a real name instead of the URL. Could you change that?

**Comment by lniehaus** [comment]
> @Anafabula unified the api docs, in the last days, your docs don't follow the new structure. Every page now hast an overview section at the beginning, https://naturerobots.github.io/HSOS-SEP-PlantMap-2022/reference/bed-plant-endpoints/#overview. And the headers after that use a real name instead of the URL. Could you change that?

I reworked the 3d-view to contain an overview like described and added a storage system section

File: docs/reference/3d-view.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+| Name                                                                | HTTP | URL                                                                               |   |   |
+|---------------------------------------------------------------------|------|-----------------------------------------------------------------------------------|---|---|
+| [Get list of 3d meta plant information](#3d-meta-plant-information-list) | GET  | `/companies/<int:company_id>/gardens/<int:garden_id>/beds/<int:bed_id>/3d-scene/` |   |   |
+
+## 3d meta plant information list
+
+Send a GET request to this endpoint to get the Plant information.
+The response contains two elements.
+Firstly an array called plants, which contains informations about every plant in the bed.
+Secondly an global elements which describes center position of the bed.
```
Comment (jarkenau) [inline docs/reference/3d-view.md:14]:
```suggestion
Secondly an global element which describes center position of the bed.
```

File: docs/reference/storage-system.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Storage System
+
+## Basics
+
+To store files like the garden images, logs or plant pointclouds we use the storage folder.
+images is a common folder, which allows to store files that are not accessible through an url.
```
Comment (jarkenau) [inline docs/reference/storage-system.md:6]:
```suggestion
Images is a common folder, which allows to store files that are not accessible through an url.
```

**Review by jarkenau — COMMENTED** [review]
typos

**Pull request merged**